Cargo.toml

```diff
@@ -9,18 +9,18 @@ version = "1.1.1"
 publish = false
 
 [dependencies]
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.64.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.67.1" }
 
-futures = { version = "0.3", features = ["compat"] }
-semver = "1.0"
-snafu = "0.8"
-strum = { version = "0.26", features = ["derive"] }
-tokio = { version = "1.29", features = ["full"] }
-tracing = "0.1"
-pin-project = "1.0"
-warp = "0.3"
-tar = "0.4"
-flate2 = "1.0"
+flate2 = "1.0.30"
```

Comment:
Nit (optional): For most operators we only use the major.minor version. I would argue that this is similar to an operator but how knows :D

Reply:
Done in 2ee2e02